### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by Yozhef**
First, let's go with the CI updates, then the dependence, then PHP.- I will be grateful for the quick reaction

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+              with:
+                  path: ~/.composer/cache/files
+                  key: ${{ matrix.php }}-${{ matrix.symfony-versions }}
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+
+            - name: Install dependencies higth
```
Comment (shadowhand) [inline .github/workflows/ci.yaml:39]: Typo?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+        name: PHP ${{ matrix.php }} ${{ matrix.description }}
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - uses: actions/cache@v2
+              with:
+                  path: ~/.composer/cache/files
+                  key: ${{ matrix.php }}-${{ matrix.symfony-versions }}
```
Comment (shadowhand) [inline .github/workflows/ci.yaml:32]: What is `symfony-versions` doing here?

**Comment by Yozhef** [comment]
@shadowhand https://github.com/Yozhef/oauth2-google/pull/2 - build is green =)

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+
+            - name: Install dependencies higth
+              run: composer install
+              if: matrix.minimum_versions == false
+
+            - name: Install dependencies lower
+              run: composer install --no-progress --no-interaction --prefer-dist
```
Comment (shadowhand) [inline .github/workflows/ci.yaml:44]:
```suggestion
              run: composer update --no-interaction --prefer-lowest
```

File: .github/workflows/static-analysis.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '7.4'
+
+            - name: Install dependencies
+              run: composer install --no-progress --no-interaction --prefer-dist
```
Comment (shadowhand) [inline .github/workflows/static-analysis.yaml:22]:
```suggestion
              run: composer install
```

**Review by shadowhand — CHANGES_REQUESTED** [review]

**Comment by shadowhand** [comment]
@Yozhef you can remove .travis.yml as part of this PR.

**Comment by shadowhand** [comment]
@Yozhef where do I see the github actions build logs?

**Comment by Yozhef** [comment]
@shadowhand in my pr in fork https://github.com/Yozhef/oauth2-google/pull/2 -> go to check - when you see all logs

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Yozhef) [reply .github/workflows/ci.yaml:39]: I'm sorry, but I didn't understand you a bit here, what do you mean?

Comment (shadowhand) [reply .github/workflows/ci.yaml:39]: `high
```suggestion
            - name: Install dependencies
```

**Review by shadowhand — APPROVED** [review]

**Comment by Yozhef** [comment]
@shadowhand for PHP 8 you need to raise the version of PHPUnit - how do i'm looking to do:
1) raise the PHPUnit version
2) raise the code style and check to PSR-12
3) then already raise the PHP version

Does this plan suit you?

**Comment by shadowhand** [comment]
@Yozhef yes it does, thank you!

**Pull request merged**